src/apps/weblib/interface.cpp

```diff
@@ -317,7 +318,7 @@ ObjectRegistry::Handle Interface::createChart()
 	    [&](const std::shared_ptr<Gfx::ICanvas> &target,
 	        GUI::Cursor cursor)
 	{
-		::setCursor(
+		::canvas_setCursor(
```

Comment:
It was your explicit request that the `cursor change` will be not part of the "canvas" functions. What has changed?

Reply:
I belive my request was to not include setCursor() in the ICanvas interface.
Now I renamed this function because I introduced a code generator where all the other generated methods called canvas_* and I didn't want to add an exception case just for this one, because it will be removed soon anyway.